includes/wikia/Wikia.php

```diff
@@ -591,7 +591,12 @@ static public function debugBacktrace($method) {
 	static public function staffForLang( $langCode ) {
 		$staffMap = WikiFactory::getVarValueByName( 'wgFounderWelcomeAuthor', Wikia::COMMUNITY_WIKI_ID );
 
-		if ( array_key_exists( $langCode, $staffMap ) ) {
+		if (
+			is_array( $staffMap ) &&
+			array_key_exists( $langCode, $staffMap ) &&
+			is_array( $staffMap[$langCode] ) &&
+			!empty( $staffMap[$langCode] )
```

Comment:
That's too much. `!empty( $staffMap[$langCode] ) && is_array( $staffMap[$langCode] )` will do perfectly. 

http://sandbox.onlinephpfunctions.com/code/a6af3979a37537d2cfbd63e495188677e0534caa